betterproto2_compiler/src/betterproto2_compiler/templates/template.py.j2

```diff
@@ -127,6 +141,10 @@ class {{ (service.py_name + "Base") | add_to_all }}(ServiceBase):
         {% endif %}
 
         raise grpclib.GRPCError(grpclib.const.Status.UNIMPLEMENTED)
+        {% if method.server_streaming %}
+        {# yielding here changes the return type from a coroutine to an async_generator #}
```

Comment:
@AdrienVannson this is a driveby but it is truly needed for grpclib to function properly